seed/README.md

```diff
@@ -0,0 +1,61 @@
+# Challenges
+
+The `seed` directory contains all the challenges that appear on the FreeCodeCamp site. 
+
+For each challenge section, there is a JSON file (fields documented below) containing its name, seed HTML, tests, and so on. 
+
+## Usage
+
+|command|description|
+|---|---|
+| `npm run test-challenges` | run all challenge tests (for each challenge JSON file, run all `tests` against all `solutions`) |
+| `npm run test` |  run all JS tests in the system, including client, server, lint and challenge tests |
+| `node seed` | parses all the challenge JSON files and saves them into MongoDB (code is inside [index.js](index.js)) |
+| `npm run commit` | interactive tool to help you build a good commit message |
+
+## Links
+
+* [Challenge Style Guide](challenge-style-guide.md)
+
+## Challenge JSON Fields
+
+* [Challenge schema](../common/models/challenge.json) - lists all of the fields inside challenge, and describes some of them
+* [Challenge template](https://gist.github.com/systimotic/e6ab8ba82222b8e0c381ab7dc2d4ff6f) - a gist made by @systimotic, replicated below
```

Comment:
Unless @systimotic has any other views we can just keep the copy in this readme.